VirtoCommerce.CatalogModule.Core/VirtoCommerce.CatalogModule.Core.csproj

```diff
@@ -0,0 +1,68 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I forgot we do not have Core project for catalog in v2. Seems we do not need to add it then.
So let's move them back.